backend/Bindings/Edna.Bindings.LtiAdvantage/Utils/LtiAdvantageExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static async Task<ClaimsPrincipal> GetValidatedLtiLaunchClaims(this HttpR
             if (!request.Form.TryGetValue("id_token", out var idTokenValue))
                 throw new NullReferenceException("No ID token is presented in the http request.");
 
-            using HttpClient client = new HttpClient();
+            using HttpClient client = new HttpClient(new EdnaExternalHttpHandler());

```

Comment:
Why are we not using `_httpClientFactory` here? While all the other things seem to maange `new` internally, having this go against the pattern seems a bit odd.

Reply:
There is some inconsistency in the existing implementation regarding the use of `_httpClientFactory`. At some places, we are creating the `HttpClient` directly, instead of going by the other approach. To stick to the existing one, I added the Handler by passing it as a parameter for the `HttpClient` constructor. Should we this as a follow-up: making use of the factory everywhere?